### PR TITLE
Add an overlay for ImageCropper

### LINKED ISSUE #419: [Request]: Preview of UI elements during custom background image cropping
### Is your request related to a problem?

No

### How is the feature request related to the problem?

_No response_

### Describe your proposed solution

Collapse 1.73.4 adds the ability to crop custom background images. It would be nice to see a UI element preview feature implemented to make it easier to ensure that the waifu's face is displayed unobstructed by UI elements in this feature.

### Describe alternatives you've considered

スクリーンショット 2024-03-06 211348
By displaying a simple silhouette of a UI element in this way on the custom background image cropping screen, it will be easy to see how the cropped image will look.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by shatyuka**
Close #419

### CODE REVIEW

File: CollapseLauncher/Assets/Images/ImageCropperOverlay/normal.png
Comment (bagusnl) [inline CollapseLauncher/Assets/Images/ImageCropperOverlay/normal.png:1]: The image doesn't seem to have the left sidebar, is it intentional or you are planning to change it after the user input on the Discussion thread?

Comment (shatyuka) [reply CollapseLauncher/Assets/Images/ImageCropperOverlay/normal.png:1]: It's intentional.

Comment (bagusnl) [reply CollapseLauncher/Assets/Images/ImageCropperOverlay/normal.png:1]: Knowing the side bar future is still up in the air, I think its fair to leave it as is for now yeah
Thanks for the clarification~

**Review by bagusnl — APPROVED** [review]
Everything seems good~

**Review by Cryotechnic — APPROVED** [review]
Looks OK for me too!

**Review by neon-nyan — APPROVED** [review]
I approve the changes

**Pull request merged**